content/en/service_management/workflows/connections.md

```diff
@@ -105,6 +106,59 @@ To learn how to restrict connection use, see [Access and Authentication][4].
 
 To connect to an arbitrary service, use the HTTP connection type. For authentication options and setup instructions, see [HTTP action][10].
 
+## Connection groups
```

Comment:
Thanks for incorporating [these changes](https://github.com/DataDog/documentation/pull/26672/files)! 
I see that the **Connection Tags** section is still left out. Is it planned to add it in a separate PR?
Connection tags are a part of this new feature and I believe it could be practical to describe them (especially their limitations) for global understanding of the group Identifier Tags, wdyt?

Reply:
I have added the section now, thanks!